ui/src/kapacitor/containers/KapacitorPage.tsx

```diff
@@ -0,0 +1,211 @@
+import React, {PureComponent, ChangeEvent} from 'react'
+import {withRouter} from 'react-router'
+
+import {Source} from 'src/types'
+
+import {
+  getKapacitor,
+  createKapacitor,
+  updateKapacitor,
+  pingKapacitor,
+} from 'src/shared/apis'
+
+import KapacitorForm from '../components/KapacitorForm'
+
+export const defaultName = 'My Kapacitor'
+export const kapacitorPort = '9092'
+
+type FlashMessage = {type: string; text: string}
+
+interface Kapacitor {
```

Comment:
Is this a `NewKapacitor`? It's super close to `Kapacitor` as defined in `types/sources`. I'm wondering if there's a way to tie the two together somehow